application/common/core.cpp

```diff
@@ -180,6 +181,39 @@ void loop()
         {
             IMEDelegateDispatcher::shared()->dispatchShowComposition(event.text.text, strlen(event.text.text), event.edit.start, event.edit.length);
         }
+        else if (event.type == SDL_CONTROLLERBUTTONDOWN)
+        {
+            //lmLogInfo(coreLogGroup, "Controller Button Down %d %d %d", event.cbutton.which, event.cbutton.button);
+            LoomGameController::getGameController(LoomGameController::getControllerIndex(event.cbutton.which))->buttonDown(event);
```

Comment:
Will this fail with a useful error or even a non-fatal warning if getControllerIndex fails? Can it fail?